lib/server/openload.py

```diff
@@ -21,17 +21,74 @@
 # */
 import re
 import util
+import urllib2
+from aadecode import AADecoder
 
-__author__ = 'Lubomir Kucera'
+__author__ = 'Jose Riha/Lubomir Kucera'
 __name__ = 'openload'
 
 
 def supports(url):
-    return re.search(r'openload\.io/embed/.+', url) is not None
+    return re.search(r'openload\.../embed/.+', url) is not None
```

Comment:
These two dots should be replaced with `\w+`. Otherwise, top-level domains like `com` will not work while invalid domains like `@$` will.